source/parameters/tracking.md

```diff
@@ -5,5 +5,15 @@
 Tracking parameters are highly program specific but it is useful to have a group
 having some standardization.
 
+The `TrackingP` parameter group contains parameters for describing the particle
+tracking.
+The components of this group are:
+```{code} yaml
+ReferenceP:
+  tracking_type: "" # [string] type of tracking
+  start_loc: ""     # [string] Where tracking starts
+  end_loc: ""       # [string] Where tracking ends
```

Comment:
TrackingP is a parameter group for individual lattice elements. If start_loc and end_loc are global parameters, these parameters should not be in TrackingP. Also if these parameters are particular to one type of simulation and not others, the parameters should be structured within a sub-parameter group.

Reply:
I'm not sure what you mean by "locations w.r.p a global origin" If you are talking about "floor" coordinates then there is `FloorP`.

Reply:
Since different programs do tracking differently I don't see how a single overall tracking method would work.

Reply:
Should one use that parameter to indicate what tracking method is used in different programs?

Reply:
@DavidSagan OK, I removed my edits of trackingP.